site/app/templates/admin/users/DownloadForm.twig

```diff
@@ -5,19 +5,19 @@
     <input type="hidden" value='{{ download_info_json }}' id="download_info_json_id">
     <div style="width:300px">
         {% for section in reg_sections %}
-            <input type="checkbox" value="{{ section.sections_registration_id }}">Registration Section {{ section.sections_registration_id }}<br>
+            <input type="checkbox" value="{{ section.sections_registration_id }}" id="{{ section.sections_registration_id }}"><label for="{{ section.sections_registration_id }}">Registration Section {{ section.sections_registration_id }}</label><br>
         {% endfor %}
-        <input type="checkbox" value="NULL">Registration Section NULL<br>
+        <input type="checkbox" value="NULL" id="NULL"><label for="NULL">Registration Section NULL</label><br>
```

Comment:
Let's make the IDs a bit more easier to remain unique on this page and add a `registration_section_` in-front of the ID. Something just rubs me the wrong way about having the IDs be `1`, `2`, ...